pkg/cosign/cosign.go

```diff
@@ -379,6 +385,33 @@ func extractDigest(imgRef string, verified []oci.Signature, log logr.Logger) (st
 	return "", fmt.Errorf("digest not found for " + imgRef)
 }
 
+func extractIssuer(imgRef string, verified []oci.Signature, log logr.Logger) (string, error) {
+	var jsonMap map[string]interface{}
+	for _, vp := range verified {
+		payload, err := vp.Payload()
+		if err != nil {
+			return "", errors.Wrap(err, "failed to get payload")
+		}
+
+		// TODO - change to using payload.SimpleContainerImage after the next Tekton release
+		if err := json.Unmarshal(payload, &jsonMap); err != nil {
```

Comment:
@Namanl2001 - lets switch to using SimpleContainerImage. We can extract the []*SimpleContainerImage list once and then retrieve data like the digest and issuer.

Reply:
See: https://github.com/sigstore/cosign/blob/10b7f9df511414cc275f58612b5c2c6cd178fd0a/cmd/cosign/cli/verify/verify.go#L214